src/controllers/user.controller.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+const { ApiError } = require('../exceptions/api.error');
+const { userService } = require('../services/user.service');
+const {
+  validateConfirmPassword,
+  validateEmail,
+  validatePassword,
+  validateUsername,
+} = require('../utils/validation');
+
+const getAllActivated = async(req, res) => {
+  const users = await userService.getAllActivated();
+
+  res.send(users.map(user => userService.normalize(user)));
```

Comment:
split it with variable for more readable
```suggestion
  res.send(...);
```

Reply:
Do it in such cases